### ISSUE TITLE
[bug] Installing QMK via MINGW64 errors out halfway through 

### ISSUE THREAD

**Issue by acpearson** [qmk/qmk_firmware#11710]
## Describe the Bug
![Capture](https://user-images.githubusercontent.com/78098642/106023010-67554580-6094-11eb-9e55-fd8abd099fd2.PNG)
![Capture2](https://user-images.githubusercontent.com/78098642/106023015-67eddc00-6094-11eb-9f17-f9654d2c0e7a.PNG)
![Capture3](https://user-images.githubusercontent.com/78098642/106023018-67eddc00-6094-11eb-8d37-650410aac582.PNG)

I was following the install instructions for windows when these errors popped up. I am very new to this, please be gentle.

Thank you
## System Information

 - Keyboard:  Razer Black widow
   - Revision (if applicable):  
 - Operating system: Windows 10

**Comment by fauxpark** [comment]
You need to run this command first:
```
pacman --needed --noconfirm --disable-download-timeout -S git mingw-w64-x86_64-toolchain mingw-w64-x86_64-python3-pip
```
If it errors out, post it here.

**Comment by acpearson** [comment]
I did. That ran successfully.
![Capture4](https://user-images.githubusercontent.com/78098642/106024346-b18af680-6095-11eb-9562-cb90c3a2af01.PNG)

**Comment by fauxpark** [comment]
Are you sure? That's not the full output.

**Comment by acpearson** [comment]
Is this the full output?
![Capture5](https://user-images.githubusercontent.com/78098642/106024918-3d9d1e00-6096-11eb-83a2-42935f158a8e.PNG)
![Capture6](https://user-images.githubusercontent.com/78098642/106024923-3e35b480-6096-11eb-91f8-303a0e8a9886.PNG)
![Capture7](https://user-images.githubusercontent.com/78098642/106024925-3e35b480-6096-11eb-8f15-2b29deabc9cc.PNG)
![Capture8](https://user-images.githubusercontent.com/78098642/106024927-3e35b480-6096-11eb-9dbf-b390cad62761.PNG)

**Comment by fauxpark** [comment]
That doesn't look like the same command I posted...

**Comment by acpearson** [comment]
You're right! I didn't see that I forgot a little bit. Let me try again.

**Comment by acpearson** [comment]
Looks like that fixed the problem!! Thank you!
